tasks/facial_landmark_detection/dataset.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import glob
+import os
+
+import numpy as np
+import torch
+from PIL import Image
+from torch.utils.data import Dataset
+
+import transforms
+
+class FaceLandmarkDataset(Dataset):
```

Comment:
Can this be merged with https://github.com/microsoft/archai/blob/main/archai/datasets/cv/face_synthetics.py#L50 ?  If not can you add auto-downloading like the FaceSyntheticsDataset does so I can use it in AML?  I would also promote this class to the `archai/datasets/cv` folder.

Reply:
Great idea. I added auto downloading. Do we need to promote it outside this task, though? It will drag in dependency on my transforms too which is undesirable.

Reply:
perhaps a subclass?

Reply:
It has too much stuff I won't need.